addons/test_mail/models/test_mail_models.py

```diff
@@ -104,6 +104,11 @@ class MailTestFull(models.Model):
     customer_id = fields.Many2one('res.partner', 'Customer', tracking=2)
     user_id = fields.Many2one('res.users', 'Responsible', tracking=1)
     umbrella_id = fields.Many2one('mail.test', tracking=True)
+    selection_field = fields.Selection([
```

Comment:
Don't modify existing models when possible. Better have a small model for corner cases. Especially this one is not intended for that kind of tests. We have models for tracking, maybe it could be extended / inherited ... could you check ?

Reply:
> We have models for tracking, maybe it could be extended / inherited ... could you check ?

I am not sure which model you were referring to exactly, so I created a new, small one that inherits from `mail.thread`. Let me know if that's any better.